sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -337,9 +338,13 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     institution.userFirstName = user?.firstName;
     institution.userLastName = user?.lastName;
 
+    const bcPrivate =
+      INSTITUTION_TYPE_BC_PRIVATE === institution.institutionType;
```

Comment:
I believe, this check should be inside controller

Reply:
Yes you are right